stdlib/public/core/UTF8Span.swift

```diff
@@ -174,10 +177,16 @@ extension UTF8Span {
 
 @available(SwiftStdlib 6.2, *)
 extension UTF8Span {
+  /// Whether the UTF8Span is empty
+  ///
+  /// - Complexity: O(1)
   public var isEmpty: Bool {
     self.count == 0
   }
 
+  /// Access the code units as a `Span`.
```

Comment:
```suggestion
  /// A UTF-8 span used to access the code units.
```

Reply:
Hmm, it's not a `UTF8Span` but a `Span`. How about just "Access the code units"?

Reply:
Sure, but it needs to start with a noun like (A span used...) not an imperative verb (Access the code units).

```suggestion
  /// A span used to access the code units.
```

I'm not completely happy with "used to access" in my rewrite, which still feels wordy.  Is there a verb you'd normally use to describe the relationship between a span and the data that the span provides access to?  Maybe "A span that contains the code units." is better?